packages/nimble-components/src/tooltip/specs/README.md

```diff
@@ -38,7 +134,12 @@ The tooltip will have a custom template based on FAST's template. In addition to
 
 ### Future Improvements
 
```

Comment:
I think we need an interactions / UX discussion on mobile and touch compatibility of tooltips as well as keyboard accessibility of tooltips.